<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AIReplayer</class>
 <widget class="QWidget" name="AIReplayer">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1347</width>
    <height>771</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>AIReplayer</string>
  </property>
  <widget class="QWidget" name="formLayoutWidget">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>0</y>
     <width>801</width>
     <height>761</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string/>
   </property>
   <layout class="QFormLayout" name="CenterLayout">
    <property name="fieldGrowthPolicy">
     <enum>QFormLayout::AllNonFixedFieldsGrow</enum>
    </property>
   </layout>
  </widget>
  <widget class="QLabel" name="AILabel1">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>70</y>
     <width>54</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string>0号AI：</string>
   </property>
  </widget>
  <widget class="QPushButton" name="ExitButton">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>31</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string>后退</string>
   </property>
  </widget>
  <widget class="QPushButton" name="HelpButton">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>10</y>
     <width>31</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string>帮助</string>
   </property>
  </widget>
  <widget class="QLabel" name="AILabel2">
   <property name="geometry">
    <rect>
     <x>1100</x>
     <y>60</y>
     <width>54</width>
     <height>12</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string>1号AI：</string>
   </property>
  </widget>
  <widget class="QComboBox" name="OpenFileComboBox1">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>70</y>
     <width>210</width>
     <height>22</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
  </widget>
  <widget class="QComboBox" name="OpenFileComboBox2">
   <property name="geometry">
    <rect>
     <x>1140</x>
     <y>50</y>
     <width>200</width>
     <height>22</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget_2">
   <property name="geometry">
    <rect>
     <x>1110</x>
     <y>80</y>
     <width>220</width>
     <height>25</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string/>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_4">
    <item>
     <widget class="QPushButton" name="OpenFileButton2">
      <property name="styleSheet">
       <string/>
      </property>
      <property name="text">
       <string>AI</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QCheckBox" name="HumanCheckBox2">
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="text">
       <string>人</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QCheckBox" name="ModeCheck2">
      <property name="text">
       <string>调试</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>100</y>
     <width>260</width>
     <height>25</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string/>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_2">
    <item>
     <widget class="QPushButton" name="OpenFileButton1">
      <property name="styleSheet">
       <string/>
      </property>
      <property name="text">
       <string>AI</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QCheckBox" name="HumanCheckBox1">
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="text">
       <string>人</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QCheckBox" name="ModeCheck1">
      <property name="text">
       <string>调试</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QPushButton" name="LoadFilePushButton">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>10</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string>加载回放文件</string>
   </property>
  </widget>
  <widget class="QPushButton" name="LoadMapPushButton">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>40</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string>加载地图</string>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget_3">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>420</y>
     <width>91</width>
     <height>62</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string/>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QRadioButton" name="Frog1RadioButton">
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="text">
       <string>0号迷雾</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QRadioButton" name="Frog2RadioButton">
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="text">
       <string>1号迷雾</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QRadioButton" name="GodVisionRadioButton">
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="text">
       <string>上帝视角</string>
      </property>
      <property name="checked">
       <bool>true</bool>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QPushButton" name="PlayPushButton">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>420</y>
     <width>71</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string>播放/暂停</string>
   </property>
   <property name="checkable">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="StopPushButton">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>420</y>
     <width>70</width>
     <height>23</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string>停止</string>
   </property>
  </widget>
  <widget class="QSlider" name="RoundSlider">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>510</y>
     <width>261</width>
     <height>22</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string/>
   </property>
   <property name="maximum">
    <number>300</number>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QLabel" name="RoundNumberLabel">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>480</y>
     <width>140</width>
     <height>40</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string>回合数</string>
   </property>
  </widget>
  <widget class="QLabel" name="SpeedSliderLabel">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>540</y>
     <width>84</width>
     <height>12</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string>播放速度</string>
   </property>
  </widget>
  <widget class="QSlider" name="SpeedSlider">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>560</y>
     <width>260</width>
     <height>22</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string/>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QLCDNumber" name="RoundLcdNumber">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>490</y>
     <width>41</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
  </widget>
  <widget class="QWidget" name="verticalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>1110</x>
     <y>110</y>
     <width>221</width>
     <height>441</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string/>
   </property>
   <layout class="QVBoxLayout" name="RightLayout"/>
  </widget>
  <widget class="QWidget" name="verticalLayoutWidget_2">
   <property name="geometry">
    <rect>
     <x>1120</x>
     <y>560</y>
     <width>211</width>
     <height>201</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string/>
   </property>
   <layout class="QVBoxLayout" name="SmallMapLayout"/>
  </widget>
  <widget class="QWidget" name="verticalLayoutWidget_3">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>130</y>
     <width>260</width>
     <height>281</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string/>
   </property>
   <layout class="QVBoxLayout" name="LeftLayout"/>
  </widget>
  <widget class="QWidget" name="verticalLayoutWidget_4">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>590</y>
     <width>251</width>
     <height>171</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string/>
   </property>
   <layout class="QVBoxLayout" name="CreateLayout"/>
  </widget>
  <widget class="QComboBox" name="ReplayComboBox">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>10</y>
     <width>111</width>
     <height>22</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
  </widget>
  <widget class="QComboBox" name="MapComboBox">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>40</y>
     <width>111</width>
     <height>22</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
  </widget>
  <widget class="QPushButton" name="OneStepButton">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>450</y>
     <width>70</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>单步调试</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
